src/EnergyPlus/HVACVariableRefrigerantFlow.cc

```diff
@@ -9887,12 +9870,14 @@ void SetAverageAirFlow(EnergyPlusData &state,
 
     } else { // terminal unit and/or fan is off
 
-        if (!state.dataHVACVarRefFlow->VRFTU(VRFTUNum).isInOASys) state.dataLoopNodes->Node(InletNode).MassFlowRate = 0.0;
+        if (!state.dataHVACVarRefFlow->VRFTU(VRFTUNum).isInOASys) {
+            state.dataLoopNodes->Node(InletNode).MassFlowRate = 0.0;
+            OnOffAirFlowRatio = 0.0;
+        }
         if (OutsideAirNode > 0) {
             state.dataLoopNodes->Node(OutsideAirNode).MassFlowRate = 0.0;
             state.dataLoopNodes->Node(AirRelNode).MassFlowRate = 0.0;
         }
-        OnOffAirFlowRatio = 0.0;
```

Comment:
The error that showed up in the branch was for VRF TU used in the OA system. When that TU was off and there was outside air flow, the OnOffAirFlowRatio variable was set to 0, which tripped an error message. This change corrected that error message. OnOffAirFlowRatio should be > 0 when mass flow rate > 0.